src/Vts/MonteCarlo/DataStructuresValidation/SimulationInputValidation.cs

```diff
@@ -79,31 +85,22 @@ private static ValidationResult ValidateTissueInput(ITissueInput tissueInput)
                      "Tissue optical properties mua, mus', n need to be non-negative",
                      "Please check optical properties");
             }
+
             if (tissueInput is MultiLayerTissueInput)
-            {
                 return MultiLayerTissueInputValidation.ValidateInput(tissueInput);

```

Comment:
Same comment as above for these, should we inline them?

Reply:
I brought this up in Visual Studio and it is suggesting switch statements for these, maybe we could leave these now but change them to switch expressions once we move fully to .NET 6?

Reply:
I used curly braces again.  
Yes!  Going to switch in .NET 6 makes sense.  Here since the expression uses "is" cannot use switch directly here yet.